doc/read-the-docs-site/plutus-doc.cabal

```diff
@@ -69,7 +69,7 @@ executable doc-doctests
     , base               >=4.9      && <5
     , bytestring
     , containers
-    , flat
+    , flat <0.6
```

Comment:
<0.5 ?

Reply:
We can _probably_ just pin this in `plutus-core`, since everything else depends on that. Not sure.